app/YtManager/settings.py

```diff
@@ -81,15 +81,19 @@
 
 DATABASES = {
     'default': {
-        'ENGINE': 'django.db.backends.postgresql',
-        'NAME': 'postgres',
-        'USER': 'postgres',
-        'HOST': 'db',
-        'PORT': 5432,
+        'ENGINE': os.getenv('YTSM_DATABASE_ENGINE', 'django.db.backends.postgresql'),
+        'NAME': os.getenv('YTSM_DATABASE_NAME', '/usr/src/app/data/db/ytmanager.db'),
```

Comment:
I have a small issue with this default path, it will work fine in docker, but it won't work properly without docker. 

Would it work to use a path relative to the BASE_DIR (the original path was `os.path.join(BASE_DIR, 'db.sqlite3')`).

Reply:
Changed it with https://github.com/bsquidwrd/ytsm/commit/c6a8d35f1b5496f921466c41d8da9fc38811e6bf